test/es-module/test-esm-import-meta-main-eval.mjs

```diff
@@ -0,0 +1,37 @@
+import { escapePOSIXShell, mustSucceed } from '../common/index.mjs';
+import child from 'node:child_process';
+
+const execOptions = '--input-type module';
+
+const script = `
+import { mustCall } from '../common/index.mjs';
```

Comment:
Let's not import this, it has a ton of side effect and we don't need `mustCall` with top-level await
```suggestion
```